NitroxServer/ConsoleCommands/HelpCommand.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using NitroxModel.DataStructures.GameLogic;
+using NitroxModel.DataStructures.Util;
+using NitroxModel.Logger;
+using NitroxServer.ConsoleCommands.Abstract;
+using NitroxServer.ConsoleCommands.Processor;
+using NitroxServer.GameLogic.Players;
+
+namespace NitroxServer.ConsoleCommands
+{
+    internal class HelpCommand : Command
+    {
+        private readonly PlayerData playerData;
+        private Dictionary<Perms, List<string>> helpTexts = new Dictionary<Perms, List<string>>();
+
+        public HelpCommand(PlayerData playerData) : base("help", Perms.PLAYER, "", "Display help about supported commands")
+        {
+            this.playerData = playerData;
+        }
+
+        public override void RunCommand(string[] args, Optional<Player> player)
+        {
+            if (player.IsPresent())
+            {
+                List<string> cmdsText = GetHelpText(playerData.GetPermissions(player.Get().Name));
+                cmdsText.ForEach(cmdText => SendServerMessageIfPlayerIsPresent(player, cmdText));
```

Comment:
Instead of sending a packet for every command should we concat the command help text with new lines in the GetText method?  Then we don't deal with lists and just have a string.

Reply:
The original implement is to concat them. but on client side there is a restriction that chatlog lines <=255 and single line char length <= 255.

Reply:
and the current chatlog box is too small to break/warp a lot. yet i will let the chatlog box that way because i think there might be a remake from the maintainer of it in the future.